Mathlib/Algebra/Order/Group/Cone.lean

```diff
@@ -91,6 +92,15 @@ def mkOfPositiveCone {S : Type u} {G : Type v}
       by apply eq_of_sub_eq_zero; simp at nba; simpa [nba] using eq_zero_of_mem_of_neg_mem nab,
     add_le_add_left := fun a b nab c => by simpa using nab }
 
+/-- Construct a `PositiveCone` from
+the set of positive elements of an existing `OrderedAddCommGroup` -/
+def mkOfOrderedAddCommGroup (G : Type*) [OrderedAddCommGroup G] : (PositiveCone G) where
```

Comment:
This should be `PositiveCone.nonneg`

Reply:
Sorry, I don't understand -- do you mean that I should rename the function?